### DISCUSSION TITLE
New version coming

### DISCUSSION THREAD

**Discussion by usinesoft** [usinesoft/Cachalot#26] [Show and tell]
Hello everybody.

I would like to share with you the last news. Very soon a new version will be released with two exciting features.

1. A new collection layout that dramatically decreases the memory consumption for “flat” objects. Like the lines in a CSV file for example
2. Web monitoring tool for a cluster of servers. Please have a look at these screen copies

![image](https://user-images.githubusercontent.com/12799638/203991349-eeed641d-2f66-4d68-9532-47cc01eaefdd.png)

![image](https://user-images.githubusercontent.com/12799638/203991614-2aed9c5a-842a-4633-b0a3-1cdea3a029c8.png)

**Comment by AntiTenzor** [comment]
That's really great news!
Especially "memory consumption". =)
